### ISSUE TITLE
Incorrect time display in charts past midnight (24:01:30... till 01:00)

### ISSUE THREAD

**Issue by aldem** [netdata/netdata#9533]
##### Bug report summary
In dashboard chart display, the time since midnight and till 01:00 is reported as 24:01:30 and so on instead of 00:01:30:
![image](https://user-images.githubusercontent.com/754722/87315387-c0102700-c524-11ea-9ff8-e94384bd3704.png)

I see it in most recent nightly (v1.23.1-77-g013c9da6) but not in v1.23.1 stable and it affects all displayed charts, not only one.

##### OS / Environment
```
Linux pve 5.4.44-2-pve #1 SMP PVE 5.4.44-2
Debian GNU/Linux 10 (buster) (Proxmox)
```

##### Netdata version

v1.23.1-77-g013c9da6

##### Component Name

web (probably)

##### Steps To Reproduce

Not sure if this is relevant but after I have installed nightly version I have deleted all dbengine data so it was a clean start.

My configuration has nothing special:
```
[global]
        history = 90000
        memory mode = dbengine
        page cache size = 512
        dbengine disk space = 2048
        access log = none

[plugins]
        ebpf = no
        ebpf_process = no
```

##### Expected behavior
Display time since midnight properly, not as 24:..

**Comment by cakrit** [comment]
I upped the prio to medium, this shouldn't be hard to fix.

**Comment by jacekkolasa** [comment]
This is related to local browser settings, timezone and language.
@aldem Do you still experience this issue? If yes, can you please share which timezone settings to your have in the Dashboard? It's under settings/locale/"Show browser local time or server time?" section

Can you also tell which browser are you using and check your detected language, by running `navigator.language` in browser console?

**Comment by aldem** [comment]
@jacekkolasa Yes, I still do (at least in 1.26 stable, didn't try nighties yet).

Indeed, the problem is probably browser related - I have no issue with Firefox but it is present in Chrome and Edge (all most recent). Language is detected as "en" (Edge) and "en-US" (Chrome). Timezone is set to server which is in turn set to CET (though this is also client's side time zone).

**Comment by jacekkolasa** [comment]
i need to prioritise it lower since i cannot recreate it and perhaps that issue doesn't cause that many problems..
